RELEASE-NOTES.md

```diff
@@ -1,7 +1,8 @@
 # About this release
 
 [This release](https://github.com/aeternity/epoch/releases/tag/v0.4.1) is focused on stability of the testnet - the public test network of nodes and introduces some API for apps usability build on top of it.
-It also introduces a backward incompatible change in the chain format.
+In this release the [Oracle WebSocket API](https://github.com/aeternity/protocol/blob/epoch-v0.4.1/oracles/oracle_ws_api.md) is now documented and can be used to interact with Oracles on the block chain.
```

Comment:
feel free to refer to tag epoch-v0.5.0 directly (optional)

Reply:
Ok, it was v0.4.1 one line above so I thought better to be consistent!